samples/add-raster-from-service/src/main/java/com/esri/arcgismaps/sample/addrasterfromservice/components/AddRasterFromServiceViewModel.kt

```diff
@@ -0,0 +1,84 @@
+/* Copyright 2025 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgismaps.sample.addrasterfromservice.components
+
+import android.app.Application
+import androidx.lifecycle.AndroidViewModel
+import androidx.lifecycle.viewModelScope
+import com.arcgismaps.LoadStatus
+import com.arcgismaps.geometry.Point
+import com.arcgismaps.geometry.SpatialReference
+import com.arcgismaps.mapping.ArcGISMap
+import com.arcgismaps.mapping.BasemapStyle
+import com.arcgismaps.mapping.Viewpoint
+import com.arcgismaps.mapping.layers.RasterLayer
+import com.arcgismaps.raster.ImageServiceRaster
+import com.arcgismaps.toolkit.geoviewcompose.MapViewProxy
+import com.esri.arcgismaps.sample.sampleslib.components.MessageDialogViewModel
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.launch
+
+/**
+ * ViewModel for the Add raster from service sample.
+ */
+class AddRasterFromServiceViewModel(application: Application) : AndroidViewModel(application) {
+
+    // Map used by the Compose MapView
+    var arcGISMap = ArcGISMap(BasemapStyle.ArcGISDarkGray).apply {
+            initialViewpoint = Viewpoint(
+                Point(-13637000.0, 4550000.0, SpatialReference.webMercator()),
+                100000.0
+            )
+        }
```

Comment:
Formatting nitpick, and suggestion to use named args here. I find it helps with readability in our docs: 

```suggestion
    var arcGISMap = ArcGISMap(BasemapStyle.ArcGISDarkGray).apply {
        initialViewpoint = Viewpoint(
            center = Point(-13637000.0, 4550000.0, SpatialReference.webMercator()),
            scale = 100000.0
        )
    }
```